doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1578,16 +1585,26 @@ Lists approved certificate authorities. The listing contains the following infor
 
 ## 11 Message Log
 
-The purpose of the message log is to provide means to prove the reception of a regular request or response message to a third party. Messages exchanged between security servers are signed and encrypted. For every regular request and response, the security server produces a complete signed and timestamped document (Associated Signature Container \[[ASiC](#Ref_ASiC)\]).
+The purpose of the message log is to provide means to prove the reception of a regular request or response message to a third party. The security server supports three options for configuring message log:
 
-Message log data is stored to the database of the security server during message exchange. According to the configuration (see [11.1](#111-changing-the-configuration-of-the-message-log)), the timestamping of the signatures of the exchanged messages is either synchronous to the message exchange process or is done asynchronously using the time period set by the X-Road governing agency.
+- Full logging
+  - The whole message including both message body and metadata is logged. The log records can be verified afterwards and they can be used as evidence.
+- Metadata logging
+  - Only metadata is logged while message body is not logged. Verifying the log records afterwards is not possible and they cannot be used as evidence.
+- No message logging
+  - Message logging is fully disabled, neither message body nor metadata is logged. No log records are generated.
+  
+Full logging and metadata logging can be configured on subsystem level. In practice, different subsystems on the same security server may use different configuration. Instead, message logging is fully disabled on a security server level. Therefore, a subsystem that requires full or metadata logging should not be registered on the same security server with a subsystem that requires fully disabling message logging.
```

Comment:
> Full logging and metadata logging can be configured on subsystem level. In practice, different subsystems on the same security server may use different configuration. Instead, message logging is fully disabled on a security server level. Therefore, a subsystem that requires full or metadata logging should not be registered on the same security server with a subsystem that requires fully disabling message logging.

This should be changed to:

> Full logging and metadata logging can be configured on security server and subsystem level. When the security server level configuration is used, the same configuration is applied to all the subsystems. Instead, when the subsystem level configuration is used, the configuration is applied to specific subsystems only. In addition, combining the security server and subsystem level configurations is also possible, e.g., set metadata logging on the security server level and enable full logging for specific subsystems only. Instead, message logging is fully disabled on a security server level. Therefore, a subsystem that requires full or metadata logging should not be registered on the same security server with a subsystem that requires fully disabling message logging.

Reply:
Changed in https://github.com/nordic-institute/X-Road/pull/1038/commits/60cbdbc51130c8ae6b3aab8e0e143df02b9f5d35